Platon/backend/test.py

```diff
@@ -1,12 +1,12 @@
 import unittest
 
-from tests.milestone_test import MilestoneTest
-from tests.issues_test import IssuesTest
-from tests.auth_system_test import LoginTest,ResetPasswordTest
-from tests.follow_test import FollowTest
-from tests.profile_management_test import ResearchInfoTests,NotificationTests
-from tests.search_engine_test import SearchHistoryTests, UserSearchTests
-from tests.upcoming_events_test import UpcomingEventsTest
+#from tests.milestone_test import MilestoneTest
+#from tests.issues_test import IssuesTest
+#from tests.auth_system_test import LoginTest,ResetPasswordTest
+#from tests.follow_test import FollowTest
+#from tests.profile_management_test import ResearchInfoTests,NotificationTests
+from tests.search_engine_test import UserSearchTests#,SearchHistoryTests
```

Comment:
did you make sure search history tests are also passed

Reply:
Actually, eğen I control the errors on Travis-CI. Only tests of User Search fails or gives error. As a result, I did not recheck them on my local environment. Did you see any error or failure on them?